industry_restaurant/data/pos_config.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="pos_config_main_seven_star_restaurant" model="pos.config">
+        <field name="name">Seven Star Restaurant - Sans Francisco</field>
```

Comment:
I don't think you need:
<field name="use_pricelist" eval="1"/>
<field name="ship_later" eval="1"/>
<field name="cash_rounding" eval="1"/>

Reply:
We need this as consultant have created a new restaurant instead of using our standard base restaurant